capstone-frontend/src/styles/Book.css

```diff
@@ -19,12 +28,25 @@
   z-index: 100;
 }
 
-.dropdown-add {
-  margin-bottom: 5px;
+.btn-margin {
+  margin-top: 10px;
+  position: relative;
+  width: 75%;
 }
 
-#dropdown-list-add {
+#button-list-add {
   position: relative;
   width: 100%;
 }
 
+.page-container {
+  text-align: center;
+  justify-content: center;
+  padding: 15px;
+  margin-top: 15px;
```

Comment:
nit: Alphabetize declarations (https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order)

Reply:
I must've missed this one. Thanks!